source/OutfitterPanel.cpp

```diff
@@ -210,17 +210,83 @@ int OutfitterPanel::DetailWidth() const
 
 int OutfitterPanel::DrawDetails(const Point &center)
 {
-	if(!selectedOutfit)
-		return 0;
-	
-	outfitInfo.Update(*selectedOutfit, player, CanSell());
-	Point offset(outfitInfo.PanelWidth(), 0.);
-	
-	outfitInfo.DrawDescription(center - offset * 1.5 - Point(0., 10.));
-	outfitInfo.DrawRequirements(center - offset * .5 - Point(0., 10.));
-	outfitInfo.DrawAttributes(center + offset * .5 - Point(0., 10.));
-	
-	return outfitInfo.MaximumHeight();
+
+	string selectedItem = "Nothing Selected";
+	const Font &font = FontSet::Get(14);
+	const Color &bright = *GameData::Colors().Get("bright");
+	const Color &dim = *GameData::Colors().Get("medium");
+	const Sprite *collapsedArrow = SpriteSet::Get("ui/collapsed");
+
+	int heightOffset = 20;
+
+	if(selectedOutfit)
+	{
+		outfitInfo.Update(*selectedOutfit, player, CanSell());
+		selectedItem = selectedOutfit->Name();
+
+		const Sprite *thumbnail = selectedOutfit->Thumbnail();
+		const Sprite *background = SpriteSet::Get("ui/outfitter selected");
+
+		float tileSize = max(thumbnail->Height(), static_cast<float>(TileSize()));
+
+		Point thumbnailCenter(center.X(), center.Y() + 20 + tileSize / 2);
+
+		Point startPoint(center.X() - INFOBAR_WIDTH / 2 + 20, center.Y() + 20 + tileSize);
+
+		double descriptionOffset = 35.;
+		Point descCenter(Screen::Right() - SIDE_WIDTH + INFOBAR_WIDTH / 2, startPoint.Y() + 20.);
+
+		// Maintenance note: This can be replaced with collapsed.contains() in C++20
+		if(!collapsed.count("description"))
+		{
+			descriptionOffset = outfitInfo.DescriptionHeight();
+			outfitInfo.DrawDescription(startPoint);
+		}
+		else
+		{
+			std::string label = "description";
+			font.Draw(label, startPoint + Point(35., 12.), dim);
+			SpriteShader::Draw(collapsedArrow, startPoint + Point(20., 20.));
+		}
+
+		// calculate the new ClickZone for the description
+		Point descDimensions(INFOBAR_WIDTH, descriptionOffset + 10.);
+		ClickZone<std::string> collapseDescription = ClickZone<std::string>(descCenter, descDimensions, std::string("description"));
+
+		// find the old zone to erase it
+		for(auto it = categoryZones.begin(); it != categoryZones.end(); ++it)
+		{
+			if(it->Value() == "description")
+			{
+				categoryZones.erase(it);
+				break;
+			}
+
+		}
+
+		// insert the new zone
+		categoryZones.emplace_back(collapseDescription);
```

Comment:
nit: Can we overwrite the existing clickzone with the new one, rather than erase and then append?

Reply:
I'm not very motivated to do this -- we'll always have to iterate anyway to check if the clickzone exists, because if it doesn't exist it needs to be added instead. I think it's easier to understand this way and since this isn't computationally expensive I don't see a reason to change the behavior.